style.css

```diff
@@ -1,518 +1,623 @@
+    left: 0;
+    right: 0;
+    background-color: white;
+  }
+
+  header .menu {
+    display: none;
+  }
+
+  .nav-link {
+    display: initial;
+  }
+
+  .nav-link ul {
+    display: flex;
+    align-items: center;
+  }
+
+  .nav-link li a {
+    font-size: 1.3rem;
+    padding-left: 5rem;
+    color: #344563;
+  }
+
+  .main-section {
+    padding: 0 10rem;
+    margin: auto;
+    background: url(image-source/Header\ bg.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: contain;
+    border-bottom-left-radius: 100px;
+    background-color: white;
+    margin-bottom: 5rem;
+    margin-top: 4rem;
+    height: 100vh;
+  }
+
+  .main-section h1 {
+    font-size: 3rem;
+  }
+
+  .main-section p {
+    font-size: 1.5rem;
+    width: 70%;
+  }
+
+  .Works {
+    width: 100%;
+    margin: auto;
+  }
+
+  .card {
+    display: flex;
+    padding: 1.5rem;
+    width: 80vw;
+    margin: 2rem auto;
+  }
+
+  .card:hover {
+    border: 1px solid #396df2;
+    box-shadow: 0 0 1px;
+    cursor: pointer;
+  }
+
+  .profile {
+    overflow: hidden;
+    width: 50%;
+    padding: 0 1.5rem 0 0;
+  }
+
+  .profile img {
+    object-fit: contain;
+  }
+
+  .card-port-description {
+    max-width: 50%;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 1.5rem 0 0 1.5rem;
+    gap: 0.5rem;
+  }
+
+  .card-port-description h3 {
+    font-size: 2rem;
+    color: #091e42;
+  }
+
+  .profile-description h5 {
+    font-size: 1.3rem;
+    font-weight: 500;
+  }
+
+  .paragraph-description p {
+    font-size: 2vmin;
+    line-height: 1.5;
+  }
+
+  .ruby {
+    display: initial;
+    white-space: nowrap;
+  }
+
+  .port-one {
+    order: 3;
+  }
+
+  .port-two {
+    order: 4;
+  }
+
+  .port-three {
+    order: 1;
+  }
+
+  .port-four {
+    order: 2;
+  }
+
+  .order-1 {
+    order: 2;
+    padding-left: 1rem;
+  }
+
+  .order-2 {
+    order: 1;
+  }
+
+  #About {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 5rem 9rem;
+    margin: auto;
+    align-items: center;
+  }
+
+  .one {
+    display: flex;
+    flex-direction: column;
+    width: 40%;
+    gap: 1rem;
+  }
+
+  .two {
+    width: 60%;
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+  }
+
+  .languages {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 1rem;
+  }
+
+  .language {
+    width: 6rem;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0.5rem;
+  }
+
+  .framework li {
+    width: 90%;
+    font-weight: 400;
+    outline: none;
+  }
+
+  .initial {
+    border-bottom: none;
+  }
+
+  .contact-section {
+    background: url(image-source/contact\ form\ background\ shapes.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-color: #6070ff;
+  }
+
+  .el {
+    display: none;
+  }
+
+  .contact-section p {
+    padding: 0 14rem;
+  }
```

Comment:
Please consider matching the width of the `paragraph` in accordance with the [Figma Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848&t=I5q8OHyZ39km2gAG-0) Currently you are using padding but the width will change when the viewport width changes and will have the width of the container subtract 14rem( padding ). It will make it more like the Template.

Reply:
Please can you explain more? What changes should I make here exactly. Is this an optional change?

Reply:
Thanks for the review @ABDULALI3468